src/components/Header.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { View, Text, StyleSheet, Button } from "react-native";
+
+export const Header = () => (
+  <View style={styles.container}>
+    <Text style={styles.left}>Cancel</Text>
+    <Text style={styles.main}>New task</Text>
+    <Text style={styles.right}>Save</Text>
+  </View>
+);
+
+const styles = StyleSheet.create({
+  container: {
+    height: 70,
+    paddingTop: 30,
+    backgroundColor: "#1F86FF",
```

Comment:
If you think this is a color you use a lot, you can move it to an object in a new file. 
Something like:
```
const Colors = {
     PRIMARY: #1F86FF,
}
```

and then:
```suggestion
    backgroundColor: Colors.PRIMARY,
```